source/endpoints.txt

```diff
@@ -270,6 +270,76 @@ includes methods that you can call to configure the response.
                "min-fresh=60"
              )
 
+.. _endpoint-return-types:
+
+Return Types
+------------
+
+By default, endpoints encode response body content in JSON.
+MongoDB stores data as BSON and uses :manual:`MongoDB Extended JSON
+</reference/mongodb-extended-json/>`, or EJSON, to
+represent types that do not exist in the JSON format.
+To translate MongoDB EJSON data into JSON responses, endpoints
+translate fields into JSON values when possible and
+omit the values of EJSON fields that have no JSON equivalent.
+
+To avoid any field value translations or omissions, configure
+your endpoint to respond with EJSON. Use the ``Accept``
```

Comment:
I'm wondering if the Return Type configuration should be first and then the Accept header. 

The way I imagined it is that the Accept header is for certain cases where you would want to have a different response than what's stated in the Return Type for an Endpoint. The first thing users should do is configure their Endpoints to return what they want most of the time, and they can use the Accept header sometimes

Reply:
Updated the ordering, this is a great point.